relayer/msgs.go

```diff
@@ -77,10 +77,9 @@ func (c *Chain) ConnInit(counterparty *PathEnd) sdk.Msg {
 // ConnTry creates a MsgConnectionOpenTry
 func (c *Chain) ConnTry(
 	counterparty *Chain,
-	height uint64,
 ) (sdk.Msg, error) {
 	clientState, clientStateProof, consensusStateProof, connStateProof,
-		proofHeight, err := counterparty.GenerateConnHandshakeProof(height)
+		proofHeight, err := counterparty.GenerateConnHandshakeProof(counterparty.MustGetLatestLightHeight() - 1)
```

Comment:
can you add a comment on the code for the -1? someone who's not familiar with the code might not have much context